_posts/2018-12-24-ApiaryUI.md

```diff
@@ -0,0 +1,131 @@
+and [Embed](https://help.apiary.io/tools/embed/). When we have those strategies fully formed
+we'll them announce on [Apiary Blog](https://blog.apiary.io/).
+
+
+### What's new in ApiaryUI
+
+ApiaryUI went through major product decisions. The concept of having three column layout stayed,
+but we reorganized the information displayed on each of them. Human Column is now designed
+to contain all the information needed for API Viewer to fully understand what the API is all about.
+Machine Columns sole responsibility is to display machine processing related information
+about one Http Transaction - Response/Request pair.
+
+Although ApiaryUI exclusively uses [API Elements](https://apielements.org/) as data format, it can also consume
+Api Description Document (ADD) formats like [API Blueprint](https://apiblueprint.org/) or [OpenAPI 2.0](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md) ([OpenAPI 3.0](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md) coming soon)
+and it parses the ADDs in Web Worker inside the browser. This means that ApiaryUI is completely
+backend-less and does not need the server as it's backend.
+
+#### Stability and predictability
+
+ApiaryUI is React/Redux SPA (Single Page Application). We've used industry standard architectures
+and patterns to produce simple and standardized codebase. As I mentioned earlier, our extensive
+tests suite guarantees the stability of the ApiaryUI and makes further development predictable. 
+
+
+#### Code highlighting
+
+ApiaryUI highlights code examples in the following languages:
+
+- Javascript
+- JSON
+- XML
+- Python
+- Ruby
+- Java
+
+#### Print support
+
+ApiaryUI is fully print friendly. You can easily print the rendered documentation directly on paper or into PDF.
+
+
+#### Accessibility
+
+ApiaryUI complies with latest accessibility standards. We care about correctness of HTML semantics and proper use of Aria attributes.
+ApiaryUI is compatible with screen readers and even specially-abled people should navigate in it with no problems.
+Full support for touch events is guaranteed.
+
+##### Keyboard navigation
+
+ApiaryUI supports keyboard navigation for people who cannot use a mouse or other pointing device, 
+or who want to use a keyboard as much as possible.
+
+
+#### Mobile First (building for the future)
+
+Mobile devices are already eclipsing desktops in adoption and internet use — that's why ApiaryUI was built from
+the ground up to support any kind of device, any size screen, with any resolution. 
+ApiaryUI behaves and looks differently on mobile and desktop resolutions to maximize the user experience. 
+
+#### Error Boundaries
+
+ApiaryUI has full support for error containment and reporting. When the ApiaryUI intercept an unexpected failure
+it will contain it and display the Error Boundary view. This view contains button for downloading
+the complete Error Report. 
+
```

Comment:
I think this deserves further explanation on why you should care. As in "this means that if there is any problem, you are still in control of your data, but you may decide to share it with us—and we'll have your complete state, which will allow us to fix bugs much faster".

Reply:
Changed. Thanks